packages/api-client/src/store/active-entities.ts

```diff
@@ -154,15 +154,23 @@ export const createActiveEntitiesStore = ({
   })
 
   /** The currently selected server in the addressBar */
-  const activeServer = computed(
-    () =>
-      servers[
-        (activeRequest.value?.selectedServerUid ||
-          activeCollection.value?.selectedServerUid ||
-          activeCollection.value?.servers[0]) ??
-          ''
-      ],
-  )
+  const activeServer = computed(() => {
+    // Request has a selected server
+    if (isDefined(activeRequest.value?.selectedServerUid)) {
```

Comment:
```suggestion
    if (activeRequest.value?.selectedServerUid) {
```
sorry one more change, we should just check like this otherwise an empty string might fall through!